src/host.rs

```diff
@@ -43,6 +43,7 @@ pub struct Node {
     element: Element,
 }
 
+#[derive(Clone)]
```

Comment:
Does this derive clone work for closures?

Reply:
No it doesn't I need to remove this, thanks for reminding me :)